cmd/permissions-checker/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+var (
+	// VERSION will be overwritten automatically by the build system
+	VERSION = "devel"
+	// ShowHidden include hidden files or directories in the search
+	ShowHidden bool
+	// Verbose print everything
+	Verbose bool
+	// Exclude some files and/or directories (RegExp)
+	Exclude string
+	// DefaultFilePerm are the default default file permissions
+	DefaultFilePerm string
+	// DefaultDirPerm are the default directory permissions
+	DefaultDirPerm string
+	// DefaultOwner is the default owner
+	DefaultOwner string
+	// DefaultGroup is the default group
+	DefaultGroup string
+	// Directory to check
+	Directory string
+)
+
+func main() {
+	var (
+		getVersion bool
+	)
+	flag.StringVar(&Directory, "dir", "/opt/bitnami", "Directory to check")
```

Comment:
Not requiring you to change it but there is no need for all the variables to be global.

Reply:
In this case is used to obtain the relative path

Reply:
Having all those variables as global makes the reading of the other file confusing, as it is referencing them. Maybe passing a Config struct or something similar